Triphouse-project/index.html

```diff
@@ -11,52 +11,73 @@
+                  <a href="#"> </a>
+                </li>
+                <li class="header__item">
+                  <a href="#"> </a>
+                </li>
+                <li class="header__burger">
+                  <a href="#"></a>
+                </li>
+              </ul>
+            </nav>
           </div>
-          <nav class="header__nav">
-            <ul class="header__list">
-              <li class="header__item">
-                <a href="#" class="header__link">Stays</a>
-              </li>
-              <li class="header__item">
-                <a href="#" class="header__link">Attractions</a>
-              </li>
-              <li class="header__item">
-                <a href="#">
-                  <img src="./img/svg/Night/Night.svg" alt="night" />
-                </a>
-              </li>
-              <li class="header__item">
-                <a href="#">
-                  <img src="./img/svg/AccountCircle/AccountCircle.svg" alt="AcccountCircle" />
-                </a>
-              </li>
-            </ul>
-          </nav>
-        </div>
-      </header>
-      <div class="content">
-        <div class="content__title">
-          <p class="content-title__text">Discover stays to live, work or just relax</p>
-          <div class="content__sbar">
-            <ul class="sbar__lables">
-              <li class="sbar-lables__destination">Your destination or hotel name</li>
-              <li class="sbar-lables__date">Check-in — Check-out</li>
-            </ul>
+        </header>
+        <h1>
+          Discover stays <br />
+          to live, work or just relax
+        </h1>
+        <form action="" class="form__desktop">
+          <div class="sbar__first">
+            <p><label for="search">Your destination or hotel name</label></p>
+            <input id="search" type="text" name="search" placeholder="New-York" />
           </div>
-          <div class="sbar__input">
-            <input type="text" class="sbar-input__destination" placeholder="New-York" />
-            <input type="date" name="check-in" id="check-in" class="sbar__input" />
-            <input type="date" name="check-out" id="check-out" class="sbar__input" />
-            <input type="text" class="sbar__input" placeholder="Children - 0, Adult - 0" />
-            <input type="button" value="Search" class="sbar__button" />
+          <div class="sbar__second">
+            <p>
+              <label for="in">Check-in</label> -
+              <label for="out">Check-out</label>
+            </p>
+            <div>
+              <input type="date" id="in" />
+              <input type="date" id="out" />
+              </div>
           </div>
-        </div>
+          <div class="sbar__third">
+            <input type="text" class="Children" placeholder="Adults - Children - Rooms" />
+          </div>
+          
+          <button>Search</button>
+        </form>
+        <form action="" class="form__mobile">
```

Comment:
![2021-04-19_22h29_55](https://user-images.githubusercontent.com/19935417/115292639-eb344080-a15e-11eb-99f4-bc7351804115.png)
You can add a white background for the form tag, which helps to hide empty space between.

Reply:
fixed